### DISCUSSION TITLE
Sayo-nara JSON

### DISCUSSION THREAD

**Discussion by AlmaHue** [Monika-After-Story/MonikaModDev#7808] [general - piano notes]
[sayonara.txt](https://github.com/Monika-After-Story/MonikaModDev/files/5776963/sayonara.txt)

Made this. Just in case someone else wants to play Sayo-nara on MAS.

Notes:
t u o u e t u t q e t e w r y r t u
o o o o 9 u u 9 o u 7 6 7 u 6 w
o p o u 9 o 9 u 7 4 e q w w 7 t
o u i y u t y e q w q w p o
o u i y u t y e q w q w p o
o u i y u t y e q w q w p o
o u i y u t y e q w q w p o

#3765

**Comment by AlmaHue** [comment]
@urmamGAE I completely forgot to add lol, here you go

t  u  o  u  e  t  u  t  q  e  t  e  w  r  y  r  t  u
o  o  o  o  9  u  u  9  o  u  7  6  7  u  6  w
o  p  o  u  9  o  9  u  7  4  e  q  w  w  7  t
o  u  i  y  u  t  y  e  q  w  q  w  p  o
o  u  i  y  u  t  y  e  q  w  q  w  p  o
o  u  i  y  u  t  y  e  q  w  q  w  p  o
o  u  i  y  u  t  y  e  q  w  q  w  p  o

**Reply by maiikowagashiniko** [reply]
I wanted to ask on where the JSON file is? Cause from far as i know there's supposed to be a json file to trigger the song?

**Reply by AlmaHue** [reply]
afaik, github is really bad with sharing .json fsr, so all the piano .json are shared in .txt and you just have to rename it after you download

**Comment by AlmaHue** [comment]
@urmamGAE Thanks for telling and sorry for the long reply! It should be fixed now.

**Comment by jmwall24** [comment]
The issue with the o's is actually the last o in line 1 is supposed to be in line 2. so line 2 does begin with 4 o's, line 1 does NOT end with an o. I notice she moves on to line 2 when i hit o there and I confirmed in the json that last o in row 1 is actually in row 2. I'm editing the description above now.

**Comment by jujus222** [comment]
i can't find the song in the Play a song menu pls help

**Reply by AlmaHue** [reply]
play the first few notes first in order to trigger the song and it will be added to the menu when you finished the song

**Reply by jujus222** [reply]
oh ok thank you!
